### PR TITLE
added clean option to assets install command

### PR DESCRIPTION
**Pull request by robinlehrmann**
| Q             | A
| ------------- | ---
| Branch?       | master
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | #24204, #23416
| License       | MIT

### CODE REVIEW

**Comment by iltar** [comment]
Should probably go into 3.4 as it's a new feature, even though I know why it "broke".

**Comment by nicolas-grekas** [comment]
@iltar this tries to remove a behavior change introduced in #23195

**Comment by nicolas-grekas** [comment]
So, ping @stof, original reported of #23177.

**Comment by nicolas-grekas** [comment]
I'd propose to reverse the flag, and submit it on 3.4, as a new feature, so that people can opt-out if they want (yes, that'd mean keeping the BC break, I know, let's live with that.)

File: src/Symfony/Bundle/FrameworkBundle/Command/AssetsInstallCommand.php [hunk 1]
```diff
@@ -58,6 +58,7 @@ protected function configure()
             ))
             ->addOption('symlink', null, InputOption::VALUE_NONE, 'Symlinks the assets instead of copying it')
             ->addOption('relative', null, InputOption::VALUE_NONE, 'Make relative symlinks')
+            ->addOption('clean', null, InputOption::VALUE_NONE, 'Remove the assets of the bundles that no longer exist')
```
Comment (xabbuh) [inline src/Symfony/Bundle/FrameworkBundle/Command/AssetsInstallCommand.php]: @nicolas-grekas suggested the opposite (i.e. having a `no-cleanup` option or something similar) which would need to be passed if you do not want to have old assets removed.

**Comment by robinlehrmann** [comment]
@xabbuh Okay, I changed it. Good point 👍

**Review by xabbuh — APPROVED** [review]

**Review by Tobion — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @robinlehrmann.

**Pull request merged**